packages/framework/src/Model/Product/ProductCachedAttributesFacade.php

```diff
@@ -61,8 +58,10 @@ public function getProductSellingPrice(Product $product)
      */
     public function getProductParameterValues(Product $product, ?string $locale = null)
     {
-        if (isset($this->parameterValuesByProductId[$product->getId()])) {
-            return $this->parameterValuesByProductId[$product->getId()];
+        $key = (string)$product->getId();
+
+        if ($this->localCacheFacade->hasItem(static::PARAMETER_VALUES_CACHE_NAMESPACE, $key)) {
```

Comment:
I prefer try-catch :innocent:

Reply:
moreover, `getOrSaveValue()` could be used here, right?

Reply:
yes, it could, but here i want left some example of usage, and on another places will be used getOrSaveValue function